core/common/src/test/java/alluxio/test/utils/CommonTestUtils.java

```diff
@@ -9,7 +9,7 @@
  * See the NOTICE file distributed with this work for information regarding copyright ownership.
  */
 
-package alluxio;
+package alluxio.test.utils;
```

Comment:
Do we want to have a `test` package? If so, should this be named `CommonUtils` since it is already in the test package?

Reply:
Sgtm. Though the class is already in test project, it would be clear from the import to see the classes are for tests.

Reply:
sure